src/prachi/Assignment_10/Calculator.java

```diff
@@ -51,10 +48,10 @@ void printTotalAnswer(int addResult, int subResult, int mulResult, int divResult
 
 	public static void main(String[] args) {
 		Calculator cal = new Calculator();
-		int addResult = cal.add(30,10); //40
-		int subResult = cal.sub(40,20); //20
-		int mulResult = cal.mul(5,2);   //10
-		int divResult = cal.div(20,2);  //10
+		int addResult = cal.add(30,10,0); //40
```

Comment:
why we are passing 3rd arguments?
previous code is right line 54 to 57 is right.
Think on this